cypress/tests/scroll-to-contest.js

```diff
@@ -1,9 +1,4 @@
-import electionSample from '../../src/data/electionSample'
-
-const clickThoughPages = [
-  ...electionSample.contests.slice(0, 20),
-  { id: 'pre-review-page' },
-]
+const clickThoughPages = new Array(21) // number of contests + pre-review screen for activation code 'VX.23.12'
```

Comment:
sneaking in an unrelated change I see??? ;) It's fine.

Reply:
No sneaky business. 😄 Seems unrelated, but I was getting an ESLint error saying there was an issue importing the `electionSample.json` file (even though Cypress runs fine). While I was troubleshooting it, I realized that since ballot style support was added, this test no longer relies upon the data in the file to determine the length of contests… so why not just manually specify the length another way and remove the import?